packages/grid/_modules_/grid/models/events/gridEventLookup.ts

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import type {
+  GridCellEditCommitParams,
+  GridCellParams,
+  GridColumnHeaderParams,
+  GridColumnOrderChangeParams,
+  GridColumnResizeParams,
+  GridColumnVisibilityChangeParams,
+  GridEditCellPropsParams,
+  GridHeaderSelectionCheckboxParams,
+  GridRowParams,
+  GridRowScrollEndParams,
+  GridRowSelectionCheckboxParams,
+  GridScrollParams,
+} from '../params';
+import type { GridFilterModel } from '../gridFilterModel';
+import type { GridSortModel } from '../gridSortModel';
+import type { GridEditRowsModel } from '../gridEditRowModel';
+import type { GridSelectionModel } from '../gridSelectionModel';
+import type { GridState } from '../gridState';
+import type { ElementSize } from '../elementSize';
+import type { MuiBaseEvent } from '../muiEvent';
+import type { GridRowId, GridRowTreeNodeConfig } from '../gridRows';
+import type { GridPreProcessingGroup } from '../../hooks/core/preProcessing';
+
+export interface GridRowEventLookup {
+  rowClick: { params: GridRowParams; event: React.MouseEvent<HTMLElement> };
+  rowDoubleClick: { params: GridRowParams; event: React.MouseEvent<HTMLElement> };
+}
+
+export interface GridColumnHeaderEventLookup {
+  columnHeaderClick: {
+    params: GridColumnHeaderParams;
+    event: React.MouseEvent<HTMLElement>;
```

Comment:
The type of the event is always `MuiEvent`. In this case: `MuiEvent<React.MouseEvent<HTMLElement>>`

Reply:
Yes, in the lookup you have the non-wrapper version of the event
That way you can type the `publishEvent` properly.

And on the listeners you have the lookup value but wrapped in `MuiEvent`.

Reply:
If it's wrapped later that's okay, but on the docs it's not consistent.

<img width="300" src="https://user-images.githubusercontent.com/42154031/141835050-1dd173f6-52af-45b7-9efc-fb1fe8dd80b9.png" alt="image" style="max-width: 100%;">

Reply:
Yes the doc should wrap them indeed

Reply:
Done